ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_bgp.py

```diff
@@ -170,6 +172,41 @@ def router_bgp(self) -> dict | None:
             if neighbors:
                 router_bgp["neighbors"] = neighbors
 
+        for interface in self.shared_utils.wan_bgp_interfaces:
+            neighbor = {
+                "ip_address": get(interface, "peer_ip"),
+                "remote_as": get(interface, "bgp.peer_as"),
```

Comment:
You have already ensured you have correct keys.
```suggestion
                "ip_address": interface["peer_ip"],
                "remote_as": interface["bgp.peer_as"],
```